### PR TITLE
updated test cases for 2.4.0 monitoring

### PR DESCRIPTION
**Pull request by jhellar**
# Description

More info: <https://issues.redhat.com/browse/INTLY-8269>

### CODE REVIEW

File: test-cases/tests/monitoring/m03-verify-number-of-pods-for-amo.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+tags:
+  - 2.4.0
+---
+
+# M03 - Verify number of pods for AMO
```
Comment (david-martin) [inline test-cases/tests/monitoring/m03-verify-number-of-pods-for-amo.md:6]: @jhellar This one may not land for 2.4 freeze. It doesn't have an assignee yet

File: test-cases/tests/monitoring/m04-verify-rhmistatus-metric.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+More info: <https://issues.redhat.com/browse/INTLY-8120>
+
+## Steps
+
+1. Login into OpenShift console as kubeadmin
+2. Navigate to `redhat-rhmi-middleware-monitoring-operator` namespace
+3. Open **Networking > Routes**
+4. Click on **Prometheus** route
+5. Type `rhmi_status` into **Expression** field and click **Execute**
+   > Should return 8 metrics, one of them should be of value `1` (indicating current status) and the rest `0`
```
Comment (david-martin) [inline test-cases/tests/monitoring/m04-verify-rhmistatus-metric.md:17]: @jhellar This has changed.

Suggested verification:

`Should return 1 metric with the stage label having a value of the current stage`

e.g.
`rhmi_status{stage="bootstrap"} 1`

File: test-cases/tests/monitoring/m03-verify-number-of-pods-for-amo.md (hunk 1 above)
Comment (jhellar) [reply test-cases/tests/monitoring/m03-verify-number-of-pods-for-amo.md:6]: I've marked the test case as targeting 2.5.0.

**Comment by jhellar** [comment]
@david-martin thanks for review, should be fixed now.

**Comment by damienomurchu** [comment]
/lgtm
(think this may be needed to merge)

**Pull request merged**